src/test/lib/optimizer/optimizer_test.cpp

```diff
@@ -218,10 +222,10 @@ TEST_F(OptimizerTest, OptimizesSubqueriesExactlyOnce) {
   class MockRule : public AbstractRule {
    public:
     explicit MockRule(size_t& init_counter) : counter(init_counter) {}
-
-    size_t& counter;
+    std::string name() const override { return "MockRule"; }
 
    protected:
+    size_t& counter;
```

Comment:
Why has this changed? If it stays like this, should it become `_counter`?

Reply:
Reverted by now.